docs/content/guides/dagster/branch_deployments.mdx

```diff
@@ -179,26 +179,30 @@ Refer to the [Dagster Cloud environment variables documentation](/dagster-cloud/
 We’ll first need to define a job that clones our `PRODUCTION` database for each branch deployment. Later, in our GitHub actions workflow, we can trigger this job to run upon each redeploy. Each clone will be named `PRODUCTION_CLONE_<ID>` with `<ID>` representing the pull request ID, ensuring each branch deployment has a unique clone. This job will drop a database clone if it exists and then reclone from production, ensuring each redeployment has a fresh clone of `PRODUCTION`:
 
 ```python file=/guides/dagster/development_to_production/branch_deployments/clone_and_drop_db.py startafter=start_clone_db endbefore=end_clone_db
-from dagster_snowflake import SnowflakeConnection
+from dagster_snowflake import SnowflakeResource
 
-from dagster import In, Nothing, ResourceParam, graph, op
+from dagster import In, Nothing, graph, op
 
 
-@op(required_resource_keys={"snowflake"})
-def drop_database_clone(snowflake: ResourceParam[SnowflakeConnection]):
-    snowflake.execute_query(
-        "DROP DATABASE IF EXISTS"
-        f" PRODUCTION_CLONE_{os.environ['DAGSTER_CLOUD_PULL_REQUEST_ID']}"
-    )
+@op
```

Comment:
Should we (at some point) explain why we're using ops over assets here? I know some users only want to run SQL queries and they've been pointed to assets, but as these don't *technically* create assets (with the exception of `create database`, I guess), ops seem appropriate. Mostly just curious.

Reply:
ok I added a little note in. I tried to keep it brief and high level, but let me know if it needs more details or should hit on other topics